cluster/gce/trusty/kube-manifests/kube-apiserver.manifest

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
why you need your own copy of kube-apiserver.manifest? same with below other .manifest files for other master components?